ent/schema/wallet.go

```diff
@@ -107,6 +107,21 @@ func (Wallet) Fields() []ent.Field {
 			),
 		field.JSON("config", types.WalletConfig{}).
 			Optional(),
+		field.JSON("alert_config", &types.AlertConfig{}).
+			SchemaType(map[string]string{
+				"postgres": "jsonb",
+			}).
+			Optional(),
+
+		field.Bool("alert_enabled").
+			Optional().
+			Default(false),
```

Comment:
this way we will have to manually update the wallets and set this flag. Lets just assume we have the low balance alert for all wallets? People will anyways have to subscribe to webhooks so it is fine.